packages/@lwc/engine-core/src/3rdparty/snabbdom/snabbdom.ts

```diff
@@ -206,6 +216,27 @@ export function updateStaticChildren(parentElm: Node, oldCh: VNodes, newCh: VNod
     }
 }
 
+export function updateFakeSlotStaticChildren(
+    startElm: Node,
+    endElm: Node,
```

Comment:
`endElm` is not used in this function, right? Can it be removed from the arguments?